react/src/components/ButtonLink.js

```diff
@@ -1,14 +1,11 @@
-import { Component } from 'react';
 import { Link } from 'react-router-dom';
 
-class Button extends Component {
-  render() {
-    return (
-      <Link {...this.props} className="btn">
-        {this.props.children}
-      </Link>
-    );
-  }
+function Button({ props }) {
```

Comment:
`props` should not be destructured with ({ props }). It should be 
`function Button(props) {`

Reply:
addressed in https://github.com/sentry-demos/empower/pull/368/commits/ff645dc059c1f9966870698159a6dd4ea99c9d3d